modules/solid_mechanics/include/materials/SolidModel.h

```diff
@@ -86,6 +86,8 @@ class SolidModel : public DerivativeMaterialInterface<Material>
   const CRACKING_RELEASE _cracking_release;
   Real _cracking_stress;
   const Real _cracking_residual_stress;
+  const Real _cracking_beta;
+  const std::string  _compute_method;
```

Comment:
Why did you make this a string now? Leave it an int. (and remove that extra space)